packages/story-editor/src/app/taxonomy/taxonomyProvider.js

```diff
@@ -145,50 +146,46 @@ function TaxonomyProvider(props) {
   );
 
   const addSearchResultsToCache = useCallback(
-    async (
```

Comment:
pulled this change in from https://github.com/google/web-stories-wp/pull/9211 to sync up on callback updates